composer.json

```diff
@@ -36,8 +36,8 @@
     "require": {
         "php": "^7.2 || ^8.0",
         "behat/transliterator": "~1.2",
-        "doctrine/annotations": "^1.2",
-        "doctrine/cache": "^1.0",
+        "doctrine/annotations": "^1.13",
+        "doctrine/cache": "^1.11 || ^2.0",
```

Comment:
This dependency could be moved to `require-dev` now. This package is fully functional without `doctrine/cache`.

Reply:
Changed, thanks for pointing this out!